test/model/model.jl

```diff
@@ -61,8 +61,8 @@ Base.@kwdef struct ModelParameters{T<:AbstractFloat}
     ny::Int = 200
     x_length::T = 400.0e3
     y_length::T = 400.0e3
-    dx::T = x_length / nx
-    dy::T = y_length / ny
+    dx::T = x_length / (nx - 1)
+    dy::T = y_length / (ny - 1)
```

Comment:
Can you remind me why this change?

Reply:
It comes down to whether we define `nx` and `ny` as being the number of cells or nodes in the grid. We had kind of implicitly defined them as the number of cells, but I think it made more sense to adapt Alex's (and Dietrich&Newsan's) definition of it being the number of nodes. Then `dx`, being the width of a cell is the total width divided by the number of cells, which is 1 less than the number of nodes.

Reply:
Ok, thanks!